db/DB.js

```diff
@@ -212,6 +212,19 @@ const DB = {
     return updatedSubscriber;
   },
 
+  async setBreachesLastShownNow(subscriber) {
+    const nowDateTime = Date.now();
+    const nowTimeStamp = nowDateTime.toISOString();
```

Comment:
Curious... I don't think this is a thing.
[`Date.now()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) returns an integer. And it looks like [`.toISOString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) is a method on the Date object.

This was giving me the following error when trying to run in a devtools console or in Node directly:

```
// const nowTimeStamp = nowDateTime.toISOString();
//                                  ^

// TypeError: nowDateTime.toISOString is not a function
//     at Object.<anonymous> (/Users/pdehaan/dev/github/mozilla/fingerprinting-cryptomining-retention-study/tmp.js:2:34)
//     at Module._compile (internal/modules/cjs/loader.js:701:30)
//     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
//     at Module.load (internal/modules/cjs/loader.js:600:32)
//     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
//     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
//     at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
//     at startup (internal/bootstrap/node.js:283:19)
//     at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

Workarounds would be:

1. Wrap the `nowDateTime` in a `new Date()` constructor:

    ```js
    const nowDateTime = Date.now();
    // const nowTimeStamp = nowDateTime.toISOString();
    const nowTimeStamp = new Date(nowDateTime).toISOString();
    ```

1. Use the Date constructor and one line it:

    ```js
    const nowTimeStamp = new Date().toISOString();
    console.log(nowTimeStamp);
    ```

Reply:
Ah yes, meant to restore `new Date()` in there somewhere.